### DISCUSSION TITLE
Extend Collection Initializers to Collections via multiple Adds or AddRange

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#4087] [Language Ideas]
It would be great if I could do:

```
var Array1 = new[]{1,2,3,4,5};
var Array2 = new[]{6,7,8,9,0};
var List3 = new List<int>(){
  Array1, Array2
};

```

**Comment by alrz** [comment]
You can use an extension method for that:
```cs
public static void Add<T>(this List<T> list, IEnumerable<T> items) => list.AddRange(items);
```

**Reply by TonyValenti** [reply]
@alrz - Yes, this is what I do today but I think that is such a common scenerio it should be natively supported.

**Comment by Unknown6656** [comment]
I think that this should be part of https://github.com/dotnet/runtime, as it does not require a language syntax change....
